SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js

```diff
@@ -219,14 +220,15 @@ define([
           const chart = new Chart(document.getElementById(id), {
             type: 'doughnut',
             data: {
-              labels: ['Active', 'Disconected', 'Never connected'],
+              labels: ['Active', 'Pending', 'Disconnected', 'Never connected' ],
               datasets: [
                 {
-                  backgroundColor: ['#46BFBD', '#F7464A', '#949FB1'],
+                  backgroundColor: ['#46BFBD', '#FFD35E', '#F7464A', '#949FB1'],
                   data: [
                     this.scope.agentsCountActive,
                     this.scope.agentsCountDisconnected,
                     this.scope.agentsCountNeverConnected,
+                    this.scope.agentsCountPending,
```

Comment:
**though**: Should the agent count for the `pending` state match with the order of labels?